src/wallet/rpcwallet.cpp

```diff
@@ -2830,6 +2928,9 @@ static UniValue listunspent(const JSONRPCRequest& request)
             "    \"witnessScript\" : \"script\" (string) witnessScript if the scriptPubKey is P2WSH or P2SH-P2WSH\n"
             "    \"spendable\" : xxx,        (bool) Whether we have the private keys to spend this output\n"
             "    \"solvable\" : xxx,         (bool) Whether we know how to spend this output, ignoring the lack of keys\n"
+            + (avoid_reuse ?
```

Comment:
I don't think we should have help output that switches on the active wallet for that command. This breaks `help listunspent`.

Reply:
Why does it break?

Reply:
You can do `help listunspent` without specifying `-rpcwallet` in order to just get the help for `listunspent` (and other commands). By making the help dependent on `-rpcwallet`, this makes the output of `help listunspent` less helpful because then the help sometimes will include `reused` and other times not. The help text should be consistent regardless of the other options specified. In general, all options and output should be displayed in the help even if some of those results will be omitted in actual command output. For example, `getaddressinfo` has a ton of extra fields that may or may not be part of the actual result, but all of them are listed in the help output.

Reply:
Ahh, I see what you're saying. Will rework into static output post-merge and/or if rebase becomes necessary.